### DISCUSSION TITLE
Compatible with Portenta Machine Control Board (STM32H747XI)?

### DISCUSSION THREAD

**Discussion by fkromer** [stm32duino/Arduino_Core_STM32#1515] [Q&A] [answered]
Would the Portenta Machine Control Board, the board drivers [Arduino_MachineControl](https://github.com/arduino-libraries/Arduino_MachineControl) and Arduino_Core_STM32 play nicely together? The MCU on the Machine Control Board is STM32H747XI. This MCU variant is not listed explicitly in the supported MCU variants but should be almost identical to the other variants listed (STM32H747AG, STM32H747AI, STM32H747IG, STM32H747II).

**Comment by fpistm** [comment, marked as answer]
Hi,

Looking at the library, answer is no as it is designed only for the portent core based on mbed:
https://github.com/arduino-libraries/Arduino_MachineControl/blob/0aee5ef5320083e44cf9982e2cf539650dd7ce97/src/Arduino_MachineControl.h#L14

As it is specified in the library.properties:
https://github.com/arduino-libraries/Arduino_MachineControl/blob/0aee5ef5320083e44cf9982e2cf539650dd7ce97/library.properties#L9
